secant/Dependencies/FeatureFlagsManager/UserDefaultsFeatureFlagsStorage.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  UserDefaultsFeatureFlagsStorage.swift
+//  secant
+//
+//  Created by Michal Fousek on 23.02.2023.
+//
+
+import Foundation
+
+typealias UserDefaultsFeatureFlagsConfigurationProvider = UserDefaultsFeatureFlagsStorage
+typealias UserDefaultsFeatureFlagsManagerCache = UserDefaultsFeatureFlagsStorage
+
+struct UserDefaultsFeatureFlagsStorage {
+    private let userDefaults = UserDefaults.standard
+
+    enum InternalError: Error {
+        case nothingStored
+        case cantDeserializeData
+    }
+
+    enum Constants {
+        static let providerKey = "feature_flags_ud_config_provider"
+        static let cacheKey = "feature_flags_ud_config_cache"
+    }
+
+    private func load(key: String) async throws -> FeatureFlagsConfiguration {
+        guard let data = userDefaults.data(forKey: key) else { throw InternalError.nothingStored }
+        do {
+            let rawFlags = try PropertyListDecoder().decode(FeatureFlagsConfiguration.RawFlags.self, from: data)
+            return FeatureFlagsConfiguration(flags: rawFlags)
```

Comment:
```suggestion
            return Configuration(flags: rawFlags)
```

Reply:
I renamed `FeatureFlagsConfiguration` to `WalletConfig`.